src/site/content/en/blog/structured-clone/index.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: Deep-copying in JavaScript using structuredClone
+authors:
+  - surma
+hero: image/i9nJGvw3SnTPH63zKOYWtI6cP5m2/e1QXRf64ylK8blZ45H1X.jpeg
+alt: Clones. Lots of clones.
+description: >
```

Comment:
To give a short teaser on the blog listing add something with the `subhead:` key here.